src/shared/bus-unit-util.c

```diff
@@ -638,8 +638,6 @@ int bus_append_unit_property_assignment(sd_bus_message *m, const char *assignmen
 
```

Comment:
comment on the commit message: the client should **not** be altered by that, I assume?